.github/workflows/ci.yaml

```diff
@@ -20,36 +20,3 @@ jobs:
         run: |
           mkdir credentials && touch credentials/github-app-backstage-humanitec-credentials.yaml
           docker image build -t backstage:latest .
-  score-humanitec:

```

Comment:
We are not able to replace this by `humctl score validate` because we don't have any `token`/`org` in this `humanitec-architecture/backstage` repo.